nautobot/circuits/templates/circuits/inc/circuit_termination.html

```diff
@@ -21,6 +21,11 @@
                     <span class="mdi mdi-trash-can-outline" aria-hidden="true"></span> Delete
                 </a>
             {% endif %}
+            {% if termination %}
+            <a href="{% url 'circuits:circuittermination' pk=termination.pk %}" class="btn btn-xs btn-default">
+                <span class="mdi mdi-cog" aria-hidden="true"></span> Detail
```

Comment:
The `mdi-cog` icon is usually used for actions involving configuration changes and such, I think. Even though they're a bit overloaded in meaning, possibly `mdi-information` or `mdi-information-outline` might be more suitable alternatives?

Reply:
I finally settled on cog since it is kind of getting to the config, and I didn't like the other icons. But the mdi-information-outline would work. We just don't have a detail button anywhere.